fastlane/jetpack_metadata/ar-SA/release_notes.txt

```diff
@@ -1,15 +0,0 @@
-مع الإصدار 17.6، يمكنك إضافة مكوِّن الصوت إلى أي موقع على ووردبريس.كوم، أيًا كانت الخطة المشترك فيها – بما في ذلك الخطة المجانية. يسهِّل مكوِّن الصوت إضافة ملفات البث الصوتي والموسيقى والصوت إلى مقالاتك وصفحاتك.
-
-لقد عملنا بجد في مناقشة تحسينات تجربة المستخدم كذلك:
-
-عندما تقوم بتعيين صفحة رئيسية إلى صفحة من دون عنوان، سترى الآن عنصرًا نائبًا "(من دون عنوان)" بدلاً من سطر فارغ، ما يمنحك إشارة لطيفة لتسمية الصفحة.
-
-لم تتمكن أبدًا من تحرير صفحات أرشيف المقالات، نظرًا إلى أنه يتم إنشاؤها تلقائيًا. في الإصدار 17.6، لن تتمكن من فتحها في المحرِّر، ما يوفِّر عليك رحلة من دون فائدة.
-
-لقد عملنا كذلك على إصلاح أشياء:
-
-لقد اعتنينا ببعض الإشعارات التي كانت تتداخل مع ورقة الإجراءات، بما في ذلك الإشعارات التي تعرض عناصر التحكم في أيقونة الموقع، والأخرى التي تعرض إزالة البدء السريع، والثالثة التي تعرض المزيد من الإجراءات في المحرِّر.
-
-وأخيرًا، تخلصنا من خطأ كان يتسبب أحيانًا في تعطل التطبيق في أثناء إدارة التعليقات، وخطأ آخر لإصلاح خطأ تسجيل الدخول إلى مواقع WordPress.org لإظهار المُضمَّن.
-
-هذا كل ما يخص الإصدار 17.6، نراك في الإصدار 17.7!
```

Comment:
Not surprising to see this and the other release notes deleted. They were manually added as part of the 17.6 finalization and didn't get removed during code freeze or beta 1 because the automation from #16764 hadn't been merged yet.